trio/socket.py

```diff
@@ -9,6 +9,62 @@
 from . import _socket
 import sys as _sys
 
+# expose all uppercase names from standardlib socket to trio.socket
+import socket as _stdlib_socket
```

Comment:
Let's move this back down next to the loop where we use it.

Reply:
Done!